examples/save_course_slides.py

```diff
@@ -0,0 +1,142 @@
+# Run with Python3
+# Saves all presentations from course in pdfs and arranges them into folders
+# BeautifulSoup is required, just do:  pip install beautifulsoup4
+
+import requests
+import sys
+import os
+from bs4 import BeautifulSoup
+import urllib.request
+import urllib.response
+
+client_id = "..."
+client_secret = "..."
+api_host = "https://stepik.org/api/"
+
+
+# get a token
+def auth():
+    auth = requests.auth.HTTPBasicAuth(client_id, client_secret)
+    response = requests.post('https://stepik.org/oauth2/token/',
+                             data={'grant_type': 'client_credentials'},
+                             auth=auth)
+    global token
+    token = response.json().get('access_token', None)
+    if not token:
+        print('Unable to authorize with provided credentials')
+        exit(1)
+
+
+# get 1 json object
+def fetch_object(obj_class, obj_id):
+    api_url = "{}/{}s/{}".format(api_host, obj_class, obj_id)
+    response = requests.get(api_url, headers={'Authorization': 'Bearer ' + token}).json()
+    return response["{}s".format(obj_class)][0]
+
+
+# get json-objects with ids
+def fetch_objects(obj_class, ids):
```

Comment:
This version doesn't not guarantee order of objects, so it's better to use other version.

Pls look for issue - https://github.com/StepicOrg/Stepik-API/pull/53

Whole function could be replaced with new version - https://github.com/StepicOrg/Stepik-API/blob/master/examples/save_course_steps.py#L32 and you will be fine